readme.adoc

```diff
@@ -21,8 +21,11 @@ but focused on the support for hyprcursor and Xcursor only.
 - Quick start
 +
 ```bash
-# 1. install the dependencies, archlinux for example
+# 1. install the dependencies 
+# for archlinux:
 pacman -Syu python librsvg xorg-xcursorgen
+# or Fedora
+dnf install librsvg2-tools xcursorgen
```

Comment:
It might be better to list dependencies for different distributions as comments:

```bash
# 1. install the dependencies
# Archlinux: pacman -Syu python librsvg xorg-xcursorgen
# Fedora: dnf install librsvg2-tools xcursorgen
```

Reply:
Did as suggested